cubic-server/command_parser/commands/Tps.cpp

```diff
@@ -0,0 +1,70 @@
+#include "Tps.hpp"
+
+#include <algorithm>
+
+#include "Chat.hpp"
+#include "Dimension.hpp"
+#include "Player.hpp"
+#include "Server.hpp"
+#include "World.hpp"
+#include "chat/Message.hpp"
+#include "logging/logging.hpp"
+
+namespace command_parser {
+
+void Tps::autocomplete(UNUSED std::vector<std::string> &args, Player *invoker) const
+{
+    if (invoker)
+        return;
+    else
+        LINFO("autocomplete time");
+}
+
+void Tps::execute(std::vector<std::string> &args, Player *invoker) const
+{
+    switch (args.size()) {
```

Comment:
Please don't use a switch case for just 3 branching.
Please also put some of that code in other functions as the execute method is getting quite big.